README.md

```diff
@@ -0,0 +1,31 @@
+imgase-back
+=====
+
+`images-back` is the webservice to store and process images.
+
+Installation
+------------
+
+Install `images-back` from sources, by running:
+
+```sh
+git clone https://github.com/PhilLar/Images-back.git
+cd short
+go install ./cmd/images-back
+```
+
```

Comment:
Let's add instructions about the initialization of the database for newcomers:
````suggestion
Database
--------
Connect to database using `psql` and run this script to create db and user:
```sql
CREATE DATABASE imagesapp;
CREATE USER images WITH PASSWORD 'secret';
GRANT ALL PRIVILEGES ON DATABASE "imagesapp" to images;
ALTER DATABASE imagesapp OWNER TO images;
```

````